power_grid_model_c/power_grid_model/include/power_grid_model/main_model.hpp

```diff
@@ -95,9 +95,13 @@ class MainModelImpl<ExtraRetrievableTypes<ExtraRetrievableType...>, ComponentLis
         static constexpr Idx sequential = -1;
 
         CalculationMethod calculation_method{CalculationMethod::default_method};
+        OptimizerType optimizer_type{OptimizerType::no_optimization};
+        OptimizerStrategy optimizer_strategy{OptimizerStrategy::any};
+
```

Comment:
Oh, I was under the impression this was intentionally left out of this PR?

Reply:
no that was #599 , which blocked work on the C API (this PR)